css-isolation/package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "css-isolation",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "./app1",
+      "./app2"
+    ]
+  },
```

Comment:
This will break the ci.

When I wrote that deps don't install in sample subdir after run yarn start command, I mean run this in the root of the repo.

But these changes will break CI, namely how it happens now, you've changed the sample but lerna don't find it cause it private and e2e tests didn't start, as you can see

<img width="1661" alt="Screenshot 2023-01-24 at 09 01 45" src="https://user-images.githubusercontent.com/55971575/214231814-139b27e1-0b37-4c57-9598-2eba65f844ef.png">

Reply:
I fixed ci issue, you were right, learna doesn't pick up private repos by default but that is configurable.